lib/sgid-client-singleton.js

```diff
@@ -0,0 +1,23 @@
+const { SgidClient } = require('@opengovsg/sgid-client')
+const config = require('../lib/config')
+
+const randomnonce = 'randomnonce'
+const scopes = process.env.SCOPES
+
+// Initialised the sgidClient object with the different environments
+const sgidClient = {}
+for (const [env, baseurl] of Object.entries(config.baseUrls)) {
+	sgidClient[env] = new SgidClient({
```

Comment:
Instead of using the `SgidClient` directly, we can import and use the `SgidService` because it encapsulates error-handling

So instead of:

```
// `sgidClient` variable
{
   dev: <SgidClient instance>
}
```

We have:
```
// renamed to `sgidService`
{
  dev: <SgidService instance>
}
```